lib/src/main/java/com/cloudbees/groovy/cps/CpsTransformer.java

```diff
@@ -115,7 +114,7 @@ public CpsTransformer() {
         super(CompilePhase.CANONICALIZATION);
     }
 
-    public void setConfiguration(@Nonnull TransformerConfiguration config) {
+    public void setConfiguration(@NonNull TransformerConfiguration config) {
```

Comment:
Seems to me that if we are migrating some classes in `lib/src/main` from JSR 305 to SpotBugs annotations, we should migrate all of them. It seems the following changes from basil#1 did not make it into the final version of this PR:

```java
diff --git a/lib/src/main/java/com/cloudbees/groovy/cps/Env.java b/lib/src/main/java/com/cloudbees/groovy/cps/Env.java
index 0555aa9..f38c3d6 100644
--- a/lib/src/main/java/com/cloudbees/groovy/cps/Env.java
+++ b/lib/src/main/java/com/cloudbees/groovy/cps/Env.java
@@ -3,9 +3,9 @@ package com.cloudbees.groovy.cps;
 import com.cloudbees.groovy.cps.impl.LocalVariableBlock;
 import com.cloudbees.groovy.cps.impl.VariableDeclBlock;
 import com.cloudbees.groovy.cps.sandbox.Invoker;
+import edu.umd.cs.findbugs.annotations.CheckForNull;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import groovy.lang.Closure;
-import javax.annotation.CheckForNull;
 import java.io.Serializable;
 import java.util.List;
 
diff --git a/lib/src/main/java/com/cloudbees/groovy/cps/impl/CallEnv.java b/lib/src/main/java/com/cloudbees/groovy/cps/impl/CallEnv.java
index ff2423e..ab8b3fd 100644
--- a/lib/src/main/java/com/cloudbees/groovy/cps/impl/CallEnv.java
+++ b/lib/src/main/java/com/cloudbees/groovy/cps/impl/CallEnv.java
@@ -7,7 +7,7 @@ import com.cloudbees.groovy.cps.Next;
 import com.cloudbees.groovy.cps.sandbox.Invoker;
 import com.google.common.collect.Maps;
 
-import javax.annotation.Nullable;
+import edu.umd.cs.findbugs.annotations.Nullable;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
diff --git a/lib/src/main/java/com/cloudbees/groovy/cps/impl/ContinuationGroup.java b/lib/src/main/java/com/cloudbees/groovy/cps/impl/ContinuationGroup.java
index d44e234..fd0c4ed 100644
--- a/lib/src/main/java/com/cloudbees/groovy/cps/impl/ContinuationGroup.java
+++ b/lib/src/main/java/com/cloudbees/groovy/cps/impl/ContinuationGroup.java
@@ -7,6 +7,7 @@ import com.cloudbees.groovy.cps.Env;
 import com.cloudbees.groovy.cps.Next;
 import static com.cloudbees.groovy.cps.impl.SourceLocation.*;
 import com.cloudbees.groovy.cps.sandbox.Invoker;
+import edu.umd.cs.findbugs.annotations.CheckReturnValue;
 import groovy.lang.GroovyCodeSource;
 import groovy.lang.GroovyShell;
 import groovy.lang.ListWithDefault;
@@ -21,7 +22,6 @@ import java.util.IdentityHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Function;
-import javax.annotation.CheckReturnValue;
 import org.codehaus.groovy.runtime.InvokerInvocationException;
 import org.codehaus.groovy.runtime.ScriptBytecodeAdapter;
 import org.codehaus.groovy.runtime.callsite.CallSite;
diff --git a/lib/src/main/java/com/cloudbees/groovy/cps/impl/CpsCallableInvocation.java b/lib/src/main/java/com/cloudbees/groovy/cps/impl/CpsCallableInvocation.java
index 9b2ed80..68fa8ef 100644
--- a/lib/src/main/java/com/cloudbees/groovy/cps/impl/CpsCallableInvocation.java
+++ b/lib/src/main/java/com/cloudbees/groovy/cps/impl/CpsCallableInvocation.java
@@ -5,12 +5,13 @@ import com.cloudbees.groovy.cps.Continuation;
 import com.cloudbees.groovy.cps.Env;
 import com.cloudbees.groovy.cps.Next;
 
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+
 import java.util.List;
 
 import static java.util.Arrays.*;
 import java.util.Collections;
 import java.util.Map;
-import javax.annotation.CheckForNull;
 
 import groovy.lang.Closure;
 import groovy.lang.GroovyObject;
diff --git a/lib/src/main/java/com/cloudbees/groovy/cps/impl/TryBlockEnv.java b/lib/src/main/java/com/cloudbees/groovy/cps/impl/TryBlockEnv.java
index 60cadc8..1f484f7 100644
--- a/lib/src/main/java/com/cloudbees/groovy/cps/impl/TryBlockEnv.java
+++ b/lib/src/main/java/com/cloudbees/groovy/cps/impl/TryBlockEnv.java
@@ -5,7 +5,7 @@ import com.cloudbees.groovy.cps.Continuation;
 import com.cloudbees.groovy.cps.Env;
 import com.cloudbees.groovy.cps.Next;
 
-import javax.annotation.Nullable;
+import edu.umd.cs.findbugs.annotations.Nullable;
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Map.Entry;
```

Reply:
Oops, good catch, added in [e623ff9](https://github.com/cloudbees/groovy-cps/pull/120/commits/e623ff9584c3f46f36192c30ec5234d8e6e6215d).